verilog/formatting/align.cc

```diff
@@ -86,6 +95,74 @@ static bool IgnorePortDeclarationPartition(
   return false;
 }
 
+static bool IgnoreActualNamedPortPartition(
+    const TokenPartitionTree& partition) {
+  const auto& uwline = partition.Value();
+  const auto token_range = uwline.TokensRange();
+  CHECK(!token_range.empty());
+  // ignore lines containing only comments
+  if (TokensAreAllComments(token_range)) return true;
+  
+  // ignore partitions belonging to preprocessing directives
+  if (IsPreprocessorKeyword(verilog_tokentype(token_range.front().TokenEnum())))
+    return true;
+  
+  // ignore wildcard connections .*
+  if (verilog_tokentype(token_range.front().TokenEnum()) == verilog_tokentype::TK_DOTSTAR) {
+    return true;
+  }
+  
+  // ignore implicit connections .aaa
+  if (verible::SymbolCastToNode(*uwline.Origin()).MatchesTag(NodeEnum::kActualNamedPort) && 
+      TokensHaveParenthesis(token_range)) {
```

Comment:
This works for now, but in the long run, I'd like logic like this to go into functions in verilog/CST.
This one would be something like:
`bool ActualNamedPortIsImplicit(const Symbol&);`
which would examine the CST node substructure directly instead of scanning/searching for `(`.